### PR TITLE
Bicaws 1606 monitoring

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

**Review by MihaiMadeT — APPROVED** [review]

File: modules/monitoring_cluster_ecs/grafana.tf [hunk 1]
```diff
@@ -0,0 +1,271 @@
+## This entire comment block will be needed later when we set grafana up to use the db
```
Comment (jaskaransarkaria) [inline modules/monitoring_cluster_ecs/grafana.tf:1]: Has this just been added? Or an old comment 🤔

File: modules/monitoring_cluster_ecs/policies/allow_alerts_notification.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "lambda.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": ""
```
Comment (jaskaransarkaria) [inline modules/monitoring_cluster_ecs/policies/allow_alerts_notification.json:10]: If the Sid is going to be an empty string is it worth just getting rid of it?

File: modules/monitoring_cluster_ecs/scripts/test_alertmanager.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+		\"severity\":\"warning\",
+		\"instance\": \"$name.example.net\"
+	},
+	\"annotations\": {
+		\"summary\": \"High latency is high!\"
+	},
+	\"generatorURL\": \"http://prometheus.int.example.net/<generating_expression>\"
+}]"
+
+echo ""
```
Comment (jaskaransarkaria) [inline modules/monitoring_cluster_ecs/scripts/test_alertmanager.sh:23]: do we need this?

File: modules/monitoring_cluster_ecs/scripts/test_alertmanager.sh [hunk 4]
```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
+name=$RANDOM
+url="https://bichard:${PASSWORD}@${ALERTMANAGER_URL}/api/v1/alerts"
+
+echo "Firing up alert $name"
+
+# change url o
```
Comment (jaskaransarkaria) [inline modules/monitoring_cluster_ecs/scripts/test_alertmanager.sh:8]: change url o?

**Review by jaskaransarkaria — APPROVED** [review]

**Pull request merged**